config/default/manager_metrics_access_filter_rbac_patch.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+      - name: manager
+        args:
+        - "--health-probe-bind-address=:8081"
```

Comment:
Is this flag actually relevant for the metrics rbac access filter? Or can we drop it form this manifest file?

Reply:
OK, we can drop it.